README.md

```diff
@@ -59,6 +59,11 @@ docker run -td --name aind --privileged -p 5900:5900 -v /lib/modules:/lib/module
 docker exec aind cat /home/user/.vnc/passwdfile
 ```
 
+```bash
+docker run -td --name aind --privileged -p 8080:8080 -e "NOVNC=1" -v /lib/modules:/lib/modules:ro aind/aind
```

Comment:
Could you add description about NOVNC

Reply:
Also curious, maybe something like "WEBMODE" rather than "NOVNC" might be easier to understand. "NOVNC" sounds like disabling VNC.